client/scripts/views/modals/webhook_settings_modal.tsx

```diff
@@ -107,7 +107,7 @@ export class WebhookSettingsModal
               {row('New reaction', [NotificationCategories.NewReaction])}
             </List>
           </div>
-          {isChain && (
+          {isChain && Object.keys(chainNotifications).length > 0 && (
```

Comment:
this isn't right, we need to check by chainNotifications because this isn't live for every chain